src/core/lib/slice/slice_intern.cc

```diff
@@ -235,8 +240,8 @@ static InternedSliceRefcount* MatchInternedSliceLocked(uint32_t hash,
   slice_shard* shard = &g_shards[SHARD_IDX(hash)];
   /* search for an existing string */
   for (s = shard->strs[idx]; s; s = s->bucket_next) {
-    if (s->hash == hash &&
-        grpc_core::InternedSlice(s).Equals(std::forward<SliceArgs>(args)...)) {
+    if (s->hash == hash && grpc_core::InternedSlice(s).operator==(
```

Comment:
Same here.

Reply:
Same breakage as above - not sure what the syntax is to get both std::forward and == playing nicely

Reply:
Done.